tests/settings/test_settings_embedders.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from meilisearch.models.index import Embedders
+
+NEW_EMBEDDERS = {
+    "default": {
+        "source": "userProvided",
+        "dimensions": 1,
+    }
+}
```

Comment:
Let's use a fixture here. The constant is probably find, but a fixture will ensure there are no issues if values get modified in a test for some reason in the future.

```suggestion
@pytest.fixture
def new_embedders():
    return {
        "default": {
            "source": "userProvided",
            "dimensions": 1,
        }
    }
```